### ISSUE TITLE
Test with Travis

### ISSUE THREAD

**Issue by blootsvoets** [sara-nl/picasclient#4]
It would be good to continuously test the code on Travis:

Make a sara-nl account on <https://travis-ci.org> (probably needs Github sara-nl admin account). There, enable picasclient. In the root create a `.travis.yml` file, with for example the following contents:
```yaml
language: python
sudo: false
python:
  - "2.7"
  - "3.3"
  - "3.4"
  - "3.5"
install:
  - pip install -U pip
  - pip install -U .
  - pip install -U ".[test]"
script:
  - nosetests --with-coverage --cover-package=picas tests
  - flake8 picas
```
Add/remove any versions of python you want to support. Remove the last line with `flake8` if the code style is not that important.

**Comment by maarten-k** [comment]
added  .travis.yml  8c6e373

**Comment by blootsvoets** [comment]
Related suggestion: add build badge to the README.md:
```markdown
[![Build Status](https://travis-ci.org/sara-nl/picasclient.svg?branch=master)](https://travis-ci.org/sara-nl/picasclient)
```
First get it to build without error though ;-)

**Comment by maarten-k** [comment]
In f1c07578e6eac9528165e90cb55955c42d2c7bf1  I was able to fix the errors with flake8 in python 2.6 (after a few commit to fix all the flake8 warnings)
